src/main/java/uk/gov/hmcts/reform/sscs/handler/SscsCaseDataHandler.java

```diff
@@ -43,7 +52,16 @@ public CallbackResponse handle(CaseResponse caseValidationResponse,
 
                 log.info("Case created with caseId {} from exception record id {}", caseId, exceptionRecordId);
 
-                roboticsHandler.handle(caseValidationResponse, caseId, eventId);
+                if (eventId.equals(caseEvent.getCaseCreatedEventId())) {
```

Comment:
Not really sure what this means? Possibly something to describe with a descriptive method name.

Reply:
Added a private method